### DISCUSSION TITLE
Waterfox G5 (Wayland) - Global Appmenu on KDE Plasma not populating ?

### DISCUSSION THREAD

**Discussion by sharshgargh** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#63] [General]
The new G5 version has two version, the one with (Wayland) does not show Global Appmenu, can this be fixed ? it works fine with the non wayland version.

**Comment by hawkeye116477** [comment]
No idea. I'm using patch made by Canonical, if they'll fix that, then I'Il also add that here.

**Reply by sharshgargh** [reply]
Thank you
